### PR TITLE
SFR-1535: Add clear filters button

### PR DESCRIPTION
**Pull request by jackiequach**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1535)

### This PR does the following:
- Adds Clear Filters button to search results page

### Testing requirements & instructions: 
-

### CODE REVIEW

File: src/components/Search/Search.tsx [hunk 1]
```diff
@@ -274,6 +274,28 @@ const SearchResults: React.FC<{
...
+                width: "100%",
+                display: {
+                  base: "block",
+                  md: "none",
+                },
+              }}
+            >
+              Clear Filters
+            </Button>
+          ) : null}
```
Comment (hokei) [inline src/components/Search/Search.tsx:298]: a shorthand: you can replace this with the logical `&&` operator https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

File: src/components/Search/Search.tsx [hunk 2]
```diff
@@ -274,6 +274,28 @@ const SearchResults: React.FC<{
...
           />
+          {searchQuery.filters.length > 0 ? (
+            <Button
+              id="clear-filters-button"
+              buttonType="secondary"
+              type="reset"
+              onClick={() => {
+                changeFilters([]);
+              }}
+              __css={{
```
Comment (hokei) [inline src/components/Search/Search.tsx:285]: If you need a version of the "secondary" button that is blue, perhaps this is worth asking if the Design System should have one or have a way of creating a blue "secondary" button.

Comment (jackiequach) [reply src/components/Search/Search.tsx:285]: spoke to Marty, this variant is in the works for Reservoir and will be included in the next release on Thursday.

**Review by hokei — APPROVED** [review]
just a small stylistic thing, but then we're good!

**Pull request merged**